app/assets/javascripts/helpers/api_routes.js

```diff
@@ -0,0 +1,18 @@
+class ApiRoutes {
```

Comment:
this is not the correct place to put this class, you should be under app/javascript and make a folder either called helpers or routes for this. This will require you to import it

Reply:
moved file to app/javascript/helpers and exported/imported APIRoutes.